src/sequences/OEIS.ts

```diff
@@ -13,35 +45,18 @@ import axios from 'axios'
  * the Flask backend.
  *
  */
-export default class OEIS extends Cached {
+export default class OEIS extends Cached<typeof paramDesc> {
     name = 'OEIS Sequence Template'
     description = 'Factory for obtaining sequences from the OEIS'
     oeisSeq = true
```

Comment:
I know you didn't put this here but as it is the only occurrence of the identifier 'oeisSeq' in the entire codebase and you are overhauling the class properties anyway, let's take the opportunity to delete it.